### DISCUSSION TITLE
포스트 프로세싱 Build 에서 에러

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-kor#438] [Issue Report (Legacy)]
안녕하세요 ! 포스트 프로세싱 사용해봤는데, Unity 에서 잘돼고있지만, Build 하면 안돼요.

`import { PostProcessLayer } from 'UnityEngine.Rendering.PostProcessing';`

이거 있으면 에러가 나와요. 없으면 앱이 잘 돼요 

![image](https://user-images.githubusercontent.com/97385930/155682823-96de8e91-f88b-4a58-b537-6ec0f4ba8d2d.png)
![image](https://user-images.githubusercontent.com/97385930/155683044-9ec4ca3a-50a5-4165-a2db-247b1b83b6b5.png)

(android studio 에서 확인했어요)

**Comment by gusriel** [comment]
#180

유니티 포스트 프로세싱은 지원이 안된다고 적혀있는데 연관이 있을까요

**Reply by Deytooh** [reply]
@gusriel 네 하지만 최신 zepeto 업데이트에서 포스트 프로세싱 추가 돼서 사용 가능하는거 아닐까요 ? ㅠㅠ...

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~
월드 패키지 업데이트 버전과, 실행하신 제페토 앱 버전, 개발 환경 정보를 적어 주시면 문제 해결에 도움이 될 수 있을 것 같습니다.

**Reply by Deytooh** [reply]
안녕하세요 ~~ 네 !!
월드 패키지 : 1.5.0
제페토 앱 : 3.10.1
쓰는 폰 : Google Pixel 5
개발 환경 : Development ( Mobile )

**Reply by Zepeto-TechEvangelist** [reply]
넵, 우선 포스트 프로세싱을 제공하는 버전은 맞습니다. 
확인해 보고 말씀드리겠습니다!

**Reply by Deytooh** [reply]
감사합니다 !! :)

**Reply by Zepeto-TechEvangelist** [reply]
Package Manager에서 Post Processing도 설치하셨나요?


<img width="789" alt="스크린샷 2022-03-02 오후 12 37 01" src="https://user-images.githubusercontent.com/100118650/156291286-64e33b51-b8cf-490b-94fe-81a233922f00.png">

**Reply by Deytooh** [reply]
네 있어요 ! 
![image](https://user-images.githubusercontent.com/97385930/156291804-000f1166-6fb8-47f9-bd47-8d5ce622b2f3.png)

**Reply by Deytooh** [reply]
@Zepeto-TechEvangelist 오. 오늘 아침에 다시 해봤지만 안됐는데 방금 다시 해봤어요...지금은 잘 돼요 ! 진짜 이상한데 지금 잘 돼고있어요... 아무것도 안했지만 ㅋㅋㅋ 답변 너무 감사합니다 !!

**Reply by Zepeto-TechEvangelist** [reply]
넵 저도 해보니까 잘 됩니다^^
